docs/install.md

```diff
@@ -99,43 +78,60 @@ your project.
 
 1. Copy the `Auth.php` and  `AuthGroups.php` from `vendor/codeigniter4/shield/src/Config/` into your project's config folder and update the namespace to `Config`. You will also need to have these classes extend the original classes. See the example below. These files contain all of the settings, group, and permission information for your application and will need to be modified to meet the needs of your site.
 
-```php
-// new file - app/Config/Auth.php
-<?php
-
-namespace Config;
+    ```php
+    // new file - app/Config/Auth.php
+    <?php
 
-// ...
-use CodeIgniter\Shield\Config\Auth as ShieldAuth;
+    namespace Config;
 
-class Auth extends ShieldAuth
-{
     // ...
-}
-```
+    use CodeIgniter\Shield\Config\Auth as ShieldAuth;
+
+    class Auth extends ShieldAuth
+    {
+        // ...
+    }
+    ```
 
 2. **Helper Setup** The `setting` helper needs to be included in almost every page. The simplest way to do this is to add it to the `BaseController::initController` method:
 
-```php
-public function initController(RequestInterface $request, ResponseInterface $response, LoggerInterface $logger)
-{
-    $this->helpers = array_merge($this->helpers, ['setting']);
+    ```php
+    public function initController(RequestInterface $request, ResponseInterface $response, LoggerInterface $logger)
+    {
+        $this->helpers = array_merge($this->helpers, ['setting']);
 
-    // Do Not Edit This Line
-    parent::initController($request, $response, $logger);
-}
-```
+        // Do Not Edit This Line
+        parent::initController($request, $response, $logger);
```

Comment:
Side note... I've always hated that we have this there. No idea why there wasn't just a dedicated initialization function.

Reply:
What do you mean?
The Controller should have another initialization method?

Reply:
Right. There's absolutely no benefit to having those parameters exposed and adding Aline that developers shouldn't touch. Call `initController()` (and make it `final`) and then add and call a blank `initialize()` method to `CodeIgniter\Controller` and extend that in `BaseController` instead. Bam: no more weird code in userland. 

Something for v5. 🤷‍♂️